ntbs-service/Startup.cs

```diff
@@ -120,7 +120,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
             if (Env.IsEnvironment("Test"))
             {
-                app.UseDeveloperExceptionPage();
+                app.UseStatusCodePagesWithReExecute("/errors/{0}");
```

Comment:
So this intercepts the error message and looks for a file in the Errors directory within pages with the status code as a name - woo middleware! And the status code is reported correctly :)

I tested this in dev too - also just noticed I need this line in production (instead of UseExceptionHandler("Error")

Reply:
Nice!
There's one more desired behaviour I think  we want to achieve I think though, which is to still show the request id to the user on Service error pages, as that's so useful for debugging and allows the end user to quote a concrete problem to helpdesk if needed. For reference, I'm talking about one of those, which are currently displayed on the error page - here `0HLR9UJM8TQRQ`:
![image-2019-11-18-09-11-47-114](https://user-images.githubusercontent.com/6727787/69525848-e2870d00-0f60-11ea-8d09-12d7ce738e43.png)
I realise now I've not recorded this on the ticket previously, I was sure I had, sorry!